recipes/chainladder/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "chainladder" %}
+{% set version = "0.7.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/chainladder-{{ version }}.tar.gz
+  sha256: 56509c799117edd2619023be21664073eda9c1ff9944f3da879dd22395ffab39
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - joblib
+    - numpy >=1.12.0
+    - pandas >=0.23.0
+    - python
+    - scikit-learn >=0.18.0
+    - scipy
+    - sparse
+    - xlcompose >=0.2.4
```

Comment:
Just note that while the CI in staged recipes will handle building `xlcompose` before trying to build this package, when the feedstock is created, this build might fail because there is a time before `xlcompose` will show up on the conda-forge CDN. If that happens we will need to restart the CI when it's available.

Reply:
understood.  Thank you for the review and help @synapticarbors.